### PR TITLE
Api v3 Get Content

### PR DESCRIPTION
**Pull request by paglias**

### CODE REVIEW

File: website/src/controllers/api-v3/content.js [hunk 1]
```diff
@@ -10,6 +22,43 @@ function walkContent (obj, lang) {
   });
 }
 
+// After the getContent route is called the first time for a certain language
+// the response is saved on disk and subsequentially served directly from there to reduce computation.
+// Example: if `cachedContentResponses.en` is true it means that the response is cached
+let cachedContentResponses = {};
+
+// Language key set to true while the cache file is being written
+let cacheBeingWritten = {};
```
Comment (crookedneighbor) [inline website/src/controllers/api-v3/content.js:31]: Good call here.

File: website/src/controllers/api-v3/content.js [hunk 2]
```diff
@@ -24,12 +73,33 @@ api.getContent = {
   method: 'GET',
   url: '/content',
   async handler (req, res) {
-    let language = req.query.language ? req.query.language.toString() : 'en';
-    let content = _.cloneDeep(common.content);
+    let language = 'en';
+    let proposedLang = req.query.language && req.query.language.toString();
+
+    if (proposedLang && _.isBoolean(cachedContentResponses[proposedLang])) {
```
Comment (crookedneighbor) [inline website/src/controllers/api-v3/content.js]: You can also do:

``` js
if (proposedLang && proposedLang in cachedContentResponses) {
```

Which I think is more readable.

In fact, I think you _could_ just do:

``` js
if (proposedLang in cachedContentResponses) {
```

But I'd want to double check that.

**Comment by crookedneighbor** [comment]
Looks reasonable to me.

File: website/src/controllers/api-v3/content.js (hunk 2 above)
Comment (paglias) [inline website/src/controllers/api-v3/content.js]: `in` works, I'll use it

**Pull request merged**